static/bidder-info/oms.yaml

```diff
@@ -0,0 +1,11 @@
+endpoint: "http://rt.marphezis.com/pbs"
+maintainer:
+  email: "prebid@onlinemediasolutions.com"
```

Comment:
@m6yf Prebid team has sent an verification email on above address. Requesting to respond email with "received" message

Reply:
received response from OMS